apis/logging/v1/cluster_log_forwarder_types.go

```diff
@@ -352,3 +373,26 @@ type ClusterLogForwarderList struct {
 func init() {
 	SchemeBuilder.Register(&ClusterLogForwarder{}, &ClusterLogForwarderList{})
 }
+
+type LimitSpec struct {
+	// Policy is used to define the action taken
```

Comment:
Given what i see in the code:
```

			if limit.Policy == loggingv1.IgnorePolicy {
				limit.MaxRecordsPerSecond = 0 // make sure we ignore any max value given with Ignore
				limit.Policy = loggingv1.DropPolicy
			}
```
The migration says that type "ignore" means to Ignore all values completely.  This makes sense since "ignore" is a variant of "drop" as evidenced by the migration.  This needs further clarification or different design in the struct IMO.  The notes here and usage of the API imply I can add a MaxRecordsPerSecond  and Ignore which is not the case.  We should either update the notes or fix the API in a way that allows defining these policy specific config bits with the policy to which it applies

Reply:
Yes, you can have `maxRecordsPerSecond` with IgnorePolicy in CLF spec but during migration it will normalize everything to drop policy. 
@alanconway Can you please take a look at the notes?

Reply:
I've been back and forth on whether we should even have ignore.
Ignore is identical to drop+maxrecords=0; my original thinking was that completely ignoring a log stream is a special case that's worth highlighting as a policy.
However it seems more and more like the special case is just causing extra work & confusion.
@jcantrill I'm willing to drop Ignore if you think that would be clearer.

Reply:
I like the idea of an "Ignore" policy for convenience but if they only way we get it is to have to "associate" a configuration property that does not apply to it, then I think it could lead to confusion. I don't know what a "better" design would look like.

Reply:
@jcantrill Should we drop ignore policy or leave it as it is for now?